doc/source/howtos/encrypt-traffic.rst

```diff
@@ -0,0 +1,237 @@
+.. _encrypt-taffic:
+
+======================
+Encrypting PXC Traffic
+======================
+
+|PXC| supports encryption for all traffic involved in cluster operation.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _client-server-encryption:
+
+Client-Server Communication
+===========================
+
+This refers to communication between client applications and cluster nodes.
+To secure client connections,
+you need to generate and use SSL keys and certificates.
+
+The following example shows :file:`my.cnf` configuration
+for server nodes and client instances to use SSL:
+
+.. code-block:: none
+
+   [mysqld]
+   ssl-ca=ca.pem
+   ssl-cert=server-cert.pem
+   ssl-key=server-key.pem
+   
+   [client]
+   ssl-ca=ca.pem
+   ssl-cert=client-cert.pem
+   ssl-key=client-key.pem
+
+For more information, see the relevant sections about SSL certificates
+in `Galera Cluster Documentation <http://galeracluster.com/documentation-webpages/sslcert.html>`_.
+and `MySQL Server Documentation <http://dev.mysql.com/doc/refman/5.7/en/creating-ssl-files-using-openssl.html>`_.
+
+SST Traffic
+===========
+
+This refers to full data transfer
+that usually occurs when a new node (JOINER) joins the cluster
+and receives data from an existing node (DONOR).
+
+For more information, see :ref:`state_snapshot_transfer`.
+
+When copying encrypted data via SST,
+the keyring must be sent over with the files for decryption.
+Make sure that the following options are set in :file:`my.cnf` on all nodes:
+
+.. code-block:: none
+
+   early-plugin-load=keyring_file.so
+   keyring-file-data=/path/to/keyring/file
+
+.. warning:: The cluster will not work if keyring configuration across nodes
```

Comment:
I would say "keyring support" instead of "keyring configuration".  The config may be different (different file locations).  But both nodes should either (1) support the keyring or (2) not support the keyring.